### PR TITLE
Replaces foot_crank component with craftable "simple crank mechanism"

### LINKED ISSUE #59213: Craftability of footcranks
### Is your feature request related to a problem? Please describe.

At the time, there is no access to foot cranks other than disassembly of cranked machines, which is limiting in wilderness scenarios (Innawoods for example)
Access to foot cranks open some  very useful crafting recipes:

- Can sealers
- Manual oil press
- Pedal powered grindstone

It would also give us more options for light vehicle crafting

### Solution you would like.

Adding a crafting recipe for foot cranks

### Describe alternatives you have considered.

Add the recipe for Innawoods only (where you can't find them in the wild)

### Additional context

"Modern" pedal powered machines were created during the 1800's
https://www.lowtechmagazine.com/2011/05/history-of-pedal-powered-machines.html
there is a short summary of different types of machines using pedals.

The first pedal powered bicycle was created in the 1850's according to wikipedia
https://en.wikipedia.org/wiki/History_of_the_bicycle

### PR DESCRIPTION
**Pull request by merrygin**
#### Summary
Content "Replaces foot_crank component with craftable "simple crank mechanism""

#### Purpose of change
Foot cranks are used as component in various recipes (not only bicycles, also manual oil press, butter churn and a crossbow). Foot cranks are not craftable. Fixes #59213

#### Describe the solution
Make a recipe for simple crank mechanisms that replace the foot crank in all applications that don't have to do with bicycles.

#### Describe alternatives you've considered
None

#### Testing
Everything seems to show up as intended.

#### Additional context
None.

### CODE REVIEW

**Comment by Maleclypse** [comment]
So something like this https://bjsm.bmj.com/content/bjsports/13/4/170.full.pdf?

**Comment by AtomicFox556** [comment]
From #42497:

> It needs steel or iron stock, a bending jig, a gear plotting device of some kind, sheet metal tools, metal files, sheet metal, and braising gear.

I think you may want the recipe to produce a distinct item that can substitute for foot crank for most crafting, but not for building a bicycle.

**Comment by merrygin** [comment]
> So something like this https://bjsm.bmj.com/content/bjsports/13/4/170.full.pdf?

The "foot crank" here mainly seems to represent any type of simple crankset, like footpedals of a bike or simple handcranks. Nothing complicated. It might be a good idea to look at that and differentiate these items, though, maybe just call it "crank", "crank mechanism" or "crankset" and make it more complicated for different applications?
But the scope of this PR was just to make that specific item craftable, since a number of items were not craftable without finding a "foot crank" item, e.g. by disassembling a bike. More of a fix maybe.

**Comment by merrygin** [comment]
> 

I don't quite get what you mean, I think. You mean also to change the crafting of foot paddles in vehicles and define a seperate item especially for the use in bicycles? If thats what you meant: I can do that, but changing bicycle construction was not the scope of that PR, just fixing a missing recipe. ^^

**Comment by Zireael07** [comment]
@merrygin Not everything is meant to be craftable. For some items, you just need some luck

**Comment by merrygin** [comment]
> 

I understand, sure! Although I don't really see why it shouldn't be possible to craft a crank for, e.g., an oil press. But I can make it innawood only then, since the problem remains there.

**Comment by AtomicFox556** [comment]
> I don't quite get what you mean, I think. You mean also to change the crafting of foot paddles in vehicles and define a seperate item especially for the use in bicycles? If thats what you meant: I can do that, but changing bicycle construction was not the scope of that PR, just fixing a missing recipe. ^^

I was thinking of just changing this proposed recipe to produce a new "simple crank mechanism" or similar item, which is usable as an alternative to foot crank for making items such as heavy crossbow, butter churn or manual oil press, but not for installing on a vehicle.

**Comment by merrygin** [comment]
> > I don't quite get what you mean, I think. You mean also to change the crafting of foot paddles in vehicles and define a seperate item especially for the use in bicycles? If thats what you meant: I can do that, but changing bicycle construction was not the scope of that PR, just fixing a missing recipe. ^^
> 
> I was thinking of just changing this proposed recipe to produce a new "simple crank mechanism" or similar item, which is usable as an alternative to foot crank for making items such as heavy crossbow, butter churn or manual oil press, but not for installing on a vehicle.

Ah ok, got it. Sure, that makes sense.

File: data/json/recipes/weapon/ranged.json [hunk 1]
```diff
@@ -1230,7 +1230,7 @@
     "components": [
       [ [ "2x4", 8 ], [ "stick", 16 ] ],
       [ [ "rope_superior_short", 2, "LIST" ] ],
-      [ [ "foot_crank", 1 ] ],
+      [ [ "simple_crank", 1 ], [ "foot_crank", 1 ] ],
```
Comment (bombasticSlacks) [inline data/json/recipes/weapon/ranged.json:1233]: I'm not convinced the simple crank could handle the pull weight. I'm also not convinced it wouldn't severely slow down usage.

Comment (bombasticSlacks) [reply data/json/recipes/weapon/ranged.json:1233]: Hmm never mind didn't realize it was forged.

**Review by bombasticSlacks — APPROVED** [review]
This looks reasonable to me. Gonna leave it for more eyes.

**Pull request merged**